[2021-12-13 12:20:35,232] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.col_process manual__2021-12-13T12:20:26.528619+00:00 [queued]>
[2021-12-13 12:20:35,245] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.col_process manual__2021-12-13T12:20:26.528619+00:00 [queued]>
[2021-12-13 12:20:35,246] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 12:20:35,246] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-13 12:20:35,246] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 12:20:35,268] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): col_process> on 2021-12-13 12:20:26.528619+00:00
[2021-12-13 12:20:35,276] {standard_task_runner.py:52} INFO - Started process 2938 to run task
[2021-12-13 12:20:35,281] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'col_process', 'manual__2021-12-13T12:20:26.528619+00:00', '--job-id', '968', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpzo_a9abt', '--error-file', '/tmp/tmpcof3rn9y']
[2021-12-13 12:20:35,283] {standard_task_runner.py:77} INFO - Job 968: Subtask col_process
[2021-12-13 12:20:35,430] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.col_process manual__2021-12-13T12:20:26.528619+00:00 [running]> on host 218df5ddc363
[2021-12-13 12:20:35,553] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=col_process
AIRFLOW_CTX_EXECUTION_DATE=2021-12-13T12:20:26.528619+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-13T12:20:26.528619+00:00
[2021-12-13 12:20:35,629] {python.py:152} INFO - Done. Returned value was: None
[2021-12-13 12:20:35,649] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=col_process, execution_date=20211213T122026, start_date=20211213T122035, end_date=20211213T122035
[2021-12-13 12:20:35,699] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-13 12:20:35,757] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
